pkg/api/norman/customization/cred/store.go

```diff
@@ -93,27 +84,6 @@ type Store struct {
 	TokenLister        v3.TokenLister
 }
 
-// Create will add an expiration timestamp in milliseconds since Unix Epoch as a CloudCredentialExpirationAnnotation if
-// the credential is a harvester credential based on kubeconfig with a configured token, and then create the credential.
-func (s *Store) Create(apiContext *types.APIContext, schema *types.Schema, data map[string]interface{}) (map[string]interface{}, error) {
-	if err := s.processHarvesterCloudCredentialExpiration(data, false); err != nil {
-		return nil, fmt.Errorf("failed to process harvester cloud credential expiration: %w", err)
-	}
-
-	return s.Store.Create(apiContext, schema, data)
```

Comment:
Are other cloud credentials still using `v3/cloudcredentials` endpoint?  We cannot remove `s.Store.Create/Update` calls yet in that case.

Reply:
It embeds the store so we aren't removing it, just going back to the default definition. If you check the blame I added the create function.